### PR TITLE
LPS-172507 Fix the flaky tests testAuthenticateByEmailAddress and testLockout

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for [subtask LPS-172507](https://issues.liferay.com/browse/LPS-172507), [LPS-166748](https://issues.liferay.com/browse/LPS-166748)

### CODE REVIEW

**Comment by drewbrokke** [comment]
@gislayne-vitorino this looks good. There is a similar failure found in the following test cases:

```
UserAccountResourceTest > testPatchUserAccount
UserAccountResourceTest > testPutUserAccount
```

Would you be willing to fix them and add them to this pull since it's all the same fix?

Thanks!

**Comment by gislayne-vitorino** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java [hunk 1]
```diff
@@ -1437,6 +1440,13 @@ private void _testPostUserAccount(Captcha captcha, boolean enableCaptcha)
 		}
 	}
 
+	//	private void threadSetUp() {
+	//		User user = UserTestUtil.addUser();
+	//
+	//		AuditRequestThreadLocal.removeAuditThreadLocal();
+	//
+	//	}
+
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java:1449]: Looks like this is left over from development?

**Review by drewbrokke — APPROVED** [review]
Overall looks good, just a couple of comments. Thanks!

**Comment by drewbrokke** [comment]
@gislayne-vitorino sorry I accidentally "Approved" when I meant to "Request Changes" 😬

File: modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java [hunk 2]
```diff
@@ -758,6 +753,15 @@ private long[] _addUsers(int numberOfUsers) throws Exception {
 		return userIds;
 	}
 
+	private User _setUp(String password) throws Exception {
```
[the remark this answers is not in the document]
Comment (gislayne-vitorino) [reply modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java:756]: Yes, sounds good. I will change it

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java (hunk 1 above)
Comment (gislayne-vitorino) [reply modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java:1449]: Yep, sorry, going to fix it

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]
Looks great, thanks @gislayne-vitorino!

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**